pkg/storaged/lvol-tabs.jsx

```diff
@@ -250,19 +267,84 @@
         return;
     }
 
+    const pvs_as_spaces = pvs_to_spaces(client, client.vgroups_pvols[vgroup.path].filter(pvol => pvol.FreeSize > 0));
+
+    let initial_pvs;
+
+    if (!has_subvols)
+        initial_pvs = pvs_as_spaces;
+    else {
+        initial_pvs = [];
+
+        // Step 1: Find the spaces that are already used for a
+        // subvolume.  If a subvolume uses more than one, prefer the
+        // one with more available space.
+        for (const sv of subvols) {
+            let sel = null;
+            for (const p in sv) {
+                for (const spc of pvs_as_spaces)
+                    if (spc.block.path == p && (!sel || sel.size < spc.size))
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17226-20230918-142947-d0c778b0-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)